app/src/main/java/de/westnordost/streetcomplete/oauth/OsmOAuthDialogFragment.java

```diff
@@ -264,9 +265,15 @@ private void applyOAuthConsumer(OAuthConsumer c)
 
 		@Override protected Void doInBackground() throws Exception
 		{
-			long userId = new UserDao(osmConnection).getMine().id;
+			// Get userDetails from remote API
+			UserDetails userDetails = new UserDao(osmConnection).getMine();
+			// Get SC preferences editor
 			SharedPreferences.Editor editor = prefs.edit();
-			editor.putLong(Prefs.OSM_USER_ID, userId);
+			// Store user id in preferences file
+			editor.putLong(Prefs.OSM_USER_ID, userDetails.id);
+			// Store user display name in preferences file
+			editor.putString(Prefs.OSM_USER_DISPLAY_NAME, userDetails.displayName);
+			// Write new preferences in files
```

Comment:
// Does thing
doThing();

This is bad style. Comments should say why something is done, not what, because that should be clear from the code already as the code should be written in a way that makes it clearly readable. Comments that explain clearly readable code are an unnecessary duplication.

In this case, please simply remove the comments.